lua/events-and-binds.md

```diff
@@ -56,8 +56,10 @@ local callback = function (...)
     local args = {...}
     local str = ''
     for i = 1, #args, 1 do
-        if i > 1 then str = str .. ' '
-        str = str .. args[i]
+        if i > 1 then
+            str = str .. ' '
+            str = str .. args[i]
```

Comment:
this shouldn't be part of the condition. the condition was to add a space between arguments after the first.